common/src/types.rs

```diff
@@ -0,0 +1,104 @@
+use std::collections::HashMap;
+use std::fmt::Display;
+
+use alloy::{
+    eips::{BlockId, BlockNumberOrTag},
+    primitives::{B256, U256},
+};
+use eyre::{eyre, Report, Result};
+use serde::{de::Error, Deserialize, Serialize};
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct Account {
+    pub balance: U256,
+    pub nonce: u64,
+    pub code_hash: B256,
+    pub code: Option<Vec<u8>>,
```

Comment:
When `Vec<u8>` is serialized/deserialized it is done as an array of numbers. We should use some other type like alloy's `Bytes` type which encodes as a hex string which is a bit more efficient.